spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -697,6 +697,7 @@ The method `BeanManager.getInterceptionFactory()` returns an `InterceptionFactor
 <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 ----
 
+If type parameter `T` is not a class non-portable behavior results.
```

Comment:
Why can't we just specifically mention interfaces?
E.g. `is not a class` -> `is an interface`
Also, there should be a comma before "non-portable" (`is not a class, non-portable behavior results.`)